hyperspy/drawing/utils.py

```diff
@@ -378,6 +379,10 @@ def plot_spectra(
     legend_picking: bool
         If true, a spectrum can be toggle on and off by clicking on
         the legended line.
+    legend_loc : str or int
+        This parameter will be control where the legend is placed on the figure
+        (when plotting with 'cascade' or 'overlap' styles) according to
+        the pyplot legend documentation <http://matplotlib.org/1.3.1/api/pyplot_api.html#matplotlib.pyplot.legend>
```

Comment:
Docstring suggestion: "This parameter controls where the legend is placed on the figure, see the docstring of pyplot.legend"